lms/urls.py

```diff
@@ -167,6 +167,11 @@
         url(r'^register$', student_account_views.login_and_registration_form,
             {'initial_mode': 'register'}, name='register_user'),
     ]
+elif settings.FEATURES['ENABLE_EXTERNAL_LOGIN_AND_REGISTRATION']:
```

Comment:
Could you add a comment that it is a code of the Raccoongang?

Reply:
Done